### PR TITLE
fix: remove warning message

### PR DESCRIPTION
**Pull request by ambermingxin**
## Checklist
- [ ] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/fleet-intelligence-agent/blob/main/CONTRIBUTING.md).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.

## Release Notes

* **Documentation**
  * Updated DEB and RPM installation verification commands to use elevated system privileges when checking the installed version information.

* **Improvements**
  * Enhanced GPU UUID validation to accept flexible formatting options while maintaining strict UUID compliance. Added comprehensive test coverage to verify GPU identifier validation functions properly across various input formats and edge cases.

### CODE REVIEW

File: docs/install-deb.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ sudo apt install ./fleetint_VERSION_arm64.deb
 Verify:
 
 ```bash
-fleetint --version
+sudo fleetint --version
 systemctl status fleetintd
```
[the remark this answers is not in the document]
Comment (ambermingxin) [reply docs/install-deb.md:63]: We need sudo access for `host.go` for the machine id. If we don't have sudo access, we will have following warning. And it's consistent with all other fleetint commands require sudo (ref: `usage.md`). 
```
{
  "level": "warn",
  "ts": "2026-04-17T20:05:59.577Z",
  "caller": "host/machine_id.go:27",
  "msg": "failed to get UUID from dmidecode, trying to read from file",
  "error": "failed to read dmidecode for uuid: exit status 1\n\noutput: \n# dmidecode 3.5\n/sys/firmware/dmi/tables/smbios_entry_point: Permission denied \nScanning /dev/mem for entry point. \nCan't read memory from /dev/mem"
}
{
  "level": "warn",
  "ts": "2026-04-17T20:05:59.579Z",
  "caller": "host/init.go:122",
  "msg": "failed to get UUID from dmidecode",
  "error": "failed to read dmidecode for uuid: exit status 1\n\noutput: \n# dmidecode 3.5\n/sys/firmware/dmi/tables/smbios_entry_point: Permission denied \nScanning /dev/mem for entry point. InCan't read memory from /dev/mem"
}
```

**Review by jingxiang-z — APPROVED** [review]
